DuckDuckGo/SettingsSubscriptionView.swift

```diff
@@ -102,12 +106,17 @@ struct SettingsSubscriptionView: View {
                                  subtitle: UserText.settingsPProITRSubTitle,
                                  action: { isShowingITP.toggle() }, isButton: true)
                 .sheet(isPresented: $isShowingITP) {
-                    SubscriptionITPView()
+                    SubscriptionITPView().onAppear(perform: {
+                        Pixel.fire(pixel: .privacyProIdentityRestorationSettings)
```

Comment:
Same as above

Reply:
I agree and I don't like it either but that pixel trigger is `when a user clicks the button to access IDTR on the settings page, after their subscription is active`, I can't find anything in the `SettingsViewModel` that would allow me to fire that pixel at the event. The `shouldShowITP` is set at any time even if the SubscriptionITPView is never presented. Any suggestion?

Reply:
This view has a single point of entry, so it would make sense to put this pixel in the initializeView() in the ITPView model.

Reply:
Done